Signal/translations/bin/auto-genstrings

```diff
@@ -4,15 +4,6 @@ set -e
 
 SSK_DIR="../SignalServiceKit/src"
 
-pushd $SSK_DIR
```

Comment:
Sorry, I didn't mean to commit this. We should definitely revisit this though - we regularly will want to update the strings off master - and we can hit this check when we're not using the SSK development pod - or a different SSK development pod (I tend to keep around two).  

IMO the right check might be:

* in the "push string" script, not the "update string" script.
* Check that no development pod is in use.
* Check that we only push from master.